### DISCUSSION TITLE
Disable Ui

### DISCUSSION THREAD

**Discussion by johnlogostini** [NeKzor/tem#2] [Q&A] [answered]
First off thanks a ton for making this mod I played this game on PS Now back in 2016 and loved it I managed to get a Steam copy and then realized it was dead so thanks for bringing it back up

I was wondering if there's a way to toggle or disable the UI header when playing?
![315440_282](https://github.com/NeKzor/tem/assets/55333109/3bc563fa-97e1-4dcf-9f6a-9522a05c3b9a)

**Comment by NeKzor** [comment, marked as answer]
You should be able to toggle it with numpad 0 because it's still hard-coded here:
https://github.com/NeKzor/tem/blob/3d1efde048226b2b696e49af0446dc108086341a/src/UI.cpp#L158

Also note that this mod is still in pre-alpha.

**Comment by johnlogostini** [comment]
Thanks that works
https://github.com/user-attachments/assets/ad7fe94c-c33a-49ba-9660-c3c908cbd1df

**Comment by RCBoylan** [comment]
NeKzor your work quickly becoming de facto way to access T:E on pc https://steamcommunity.com/sharedfiles/filedetails/?id=3053652727&tscn=1739356154
Bravo, and thank you!

**Comment by johnlogostini** [comment]
yeah nice work dude I will be testing this on Linux soon
